test/extensions/filters/network/common/fuzz/uber_per_readfilter.cc

```diff
@@ -0,0 +1,97 @@
+#include "envoy/extensions/filters/network/direct_response/v3/config.pb.h"
+#include "envoy/extensions/filters/network/local_ratelimit/v3/local_rate_limit.pb.h"
+
+#include "extensions/filters/network/common/utility.h"
+#include "extensions/filters/network/well_known_names.h"
+
+#include "test/extensions/filters/common/ext_authz/test_common.h"
+#include "test/extensions/filters/network/common/fuzz/uber_readfilter.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+
```

Comment:
Set a file level constant like this:

namespace {
static const int SecondsPerDay = ...;
} // namespace